VideoColorizer.ipynb

```diff
@@ -83,9 +100,73 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[twitter] 1116751583386034176: Downloading webpage\n",
+      "[twitter:card] 1116751583386034176: Downloading webpage\n",
+      "[twitter:card] 1116751583386034176: Downloading guest token\n",
+      "[twitter:card] 1116751583386034176: Downloading JSON metadata\n",
+      "[download] Destination: video/source/DogShy1926.mp4\n",
+      "[download] 100% of 227.58KiB in 00:0040MiB/s ETA 00:000\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "    <div>\n",
+       "        <style>\n",
+       "            /* Turns off some styling */\n",
+       "            progress {\n",
+       "                /* gets rid of default border in Firefox and Opera. */\n",
+       "                border: none;\n",
+       "                /* Needs to be in here for Safari polyfill so background images work as expected. */\n",
+       "                background-size: auto;\n",
+       "            }\n",
+       "            .progress-bar-interrupted, .progress-bar-interrupted::-webkit-progress-bar {\n",
+       "                background: #F44336;\n",
+       "            }\n",
+       "        </style>\n",
+       "      <progress value='38' class='' max='38', style='width:300px; height:20px; vertical-align: middle;'></progress>\n",
+       "      100.00% [38/38 06:05<00:00]\n",
+       "    </div>\n",
+       "    "
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Video created here: video/result/DogShy1926.mp4\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<video alt=\"test\" autoplay \n",
```

Comment:
There's a video embed into the notebook! This should not be here I believe.

Reply:
This is the result of the prediction, I will delete it

Reply:
Done